utils/controleval.py

```diff
@@ -64,8 +65,8 @@ def calculate_stats(ctrls):
     applicablelist = ctrls - ctrllist[controls.Status.NOT_APPLICABLE]
     assessedlist = set().union(ctrllist[controls.Status.AUTOMATED]).union(ctrllist[controls.Status.SUPPORTED])\
         .union(ctrllist[controls.Status.DOCUMENTATION]).union(ctrllist[controls.Status.INHERENTLY_MET])\
-        .union(ctrllist[controls.Status.PARTIAL])
-    print("Missing:", ", ".join(sorted(c.id for c in applicablelist - assessedlist)))
+        .union(ctrllist[controls.Status.PARTIAL]).union(ctrllist[controls.Status.MANUAL])
+    print("Missing:", ", ".join(sorted(str(c.id) for c in applicablelist - assessedlist)))
```

Comment:
I forgot to mention that the change in line 69 fixes an issue when processing CIS control files:
```
python3 utils/controleval.py stats  -i cis_rhel7 -l l2_server -p rhel7
Total controls = 248
Applicable = 100.0% -- 248 / 248
Assessed = 70.97% -- 176 / 248

Automated = 63.71% -- 158 / 248
Supported = 0.0% -- 0 / 248
Documentation = 0.0% -- 0 / 248
Inherently Met = 0.0% -- 0 / 248
Partial = 7.26% -- 18 / 248
Traceback (most recent call last):
  File "/home/wsato/git/content/utils/controleval.py", line 123, in <module>
    main()
  File "/home/wsato/git/content/utils/controleval.py", line 119, in main
    subcmds[args.subcmd](controls_manager, args)
  File "/home/wsato/git/content/utils/controleval.py", line 82, in stats
    calculate_stats(ctrls)
  File "/home/wsato/git/content/utils/controleval.py", line 68, in calculate_stats
    print("Missing:", ", ".join(sorted(c.id for c in applicablelist - assessedlist)))
TypeError: '<' not supported between instances of 'float' and 'str'

```